### ISSUE TITLE
Pressing Minus Key to Hide Item Not Working

### ISSUE THREAD

**Issue by chevcheli0s** [dakanji/RefindPlus#179]
### RefindPlus Version

v0.14.0.AC Release

### Device Type

UEFI PC

### Problem Description

Can't hide item from menu.
Receive error "Error: 'Already Started' in 'AddToHiddenTags'!!"

### Problem Point

Before starting a loader or tool

### Affected Items

All OS Loaders (First Row Items)

### Debug Log

[24c12n1715.log](https://github.com/dakanji/RefindPlus/files/14236730/24c12n1715.log)

**Comment by dakanji** [comment]
I don't see an attempt to hide an item in the attached log.
Perhaps the wrong log file was attached ... seems to be the same file as your other issue.

**Comment by chevcheli0s** [comment]
@dakanji  line #589
Also I can't get screenshot with that error

**Comment by dakanji** [comment]
Seen it. Was scanning the log from the bottom up instead of top down. Sorry.

Well, this is a known issue as I noticed it myself recently. Was hoping I could get more info from another instance. 
Can you confirm whether it works with rEFind and which loader you were trying to hide?

Thanks

**Comment by chevcheli0s** [comment]
Yes, this works in rEFind. I try to hide item with grub loader.

**Comment by dakanji** [comment]
Can you run the `Hidden Entries` tool and share a screenshot?

**Comment by chevcheli0s** [comment]
![ScreenShot_000](https://github.com/dakanji/RefindPlus/assets/48658076/ba2dbf94-e422-4497-ac0b-05c436a389d3)
```
sudo blkid
/dev/nvme0n1p3: UUID="03a50d2d-fce0-4f8e-ab6c-9b4f509307f8" UUID_SUB="630aeae1-8cf9-4eb0-8149-ec6a2f42e5dd" BLOCK_SIZE="4096" TYPE="btrfs" PARTUUID="200c7770-a698-474e-8f94-a49f7007c11a"
/dev/nvme0n1p1: UUID="59DB-DCD0" BLOCK_SIZE="512" TYPE="vfat" PARTLABEL="EFI system partition" PARTUUID="8363f9d1-4b9f-4fdd-9722-b36eee6bf944"
/dev/nvme0n1p2: LABEL="Windows 11" BLOCK_SIZE="512" UUID="CC160CD0160CBD88" TYPE="ntfs" PARTLABEL="Basic data partition" PARTUUID="b986e491-a315-4d0b-8dee-72eda4446714"
/dev/sdb1: LABEL="data" UUID="b6d69731-0e3a-407c-a447-62bc3dfe7350" UUID_SUB="4d37952c-fa21-4a51-9690-b482c1a130c3" BLOCK_SIZE="4096" TYPE="btrfs" PARTUUID="6ccec5c4-01"
/dev/sda1: LABEL="backup" UUID="e28311a9-84f5-4830-aa9f-01117d8557b5" BLOCK_SIZE="4096" TYPE="ext4" PARTUUID="0828d003-01"
```

**Comment by dakanji** [comment]
Thanks.

A fix is in the pipeline and will appear in the next release.
You will have to wait for this or build your own when it is pushed into the codebase here.
You can try adding `grubx64.efi` to the `dont_scan_files` list in the interim.

As an aside, you might want to load an `EXT4` driver as this may help with the `Unknown Volume` instances.

**Comment by chevcheli0s** [comment]
@dakanji Thanks! I tried to load all fs drivers, but this volume is btrfs. And it always marked as unknown.

**Comment by dakanji** [comment]
Are you running the btrfs driver provided with the RefindPlus package?
If not, try this to see if it makes a difference.

**Comment by chevcheli0s** [comment]
@dakanji I have tried both. No difference.
Maybe this because option `also_scan_dirs @rootfs/boot`?

**Comment by dakanji** [comment]
The also scan option does not affect volume names.
I believe rEFInd could also not get a proper name and had it as `253 GiB Btrfs volume`
I will take another look at the code in this section later.

Thanks

**Comment by dakanji** [comment]
Addressed in Commit 909c6f87841c4ccb95a62db26303aa37922c07e0
